src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const myList = document.querySelector('.tree');
+
+const listItems = myList.getElementsByTagName('li');
```

Comment:
Consider using `querySelectorAll` instead of `getElementsByTagName` to select list items with a specific class.